.pre-commit-config.yaml

```
@@ -1,9 +1,26 @@
 repos:
-  - repo: https://github.com/ambv/black
+  - repo: https://github.com/psf/black
     rev: 22.3.0
     hooks:
       - id: black
 
+  - repo: https://github.com/asottile/yesqa
+    rev: v1.3.0
+    hooks:
+      - id: yesqa
+        additional_dependencies: &flake8_deps
+          - flake8-broken-line==0.4.0
+          - flake8-bugbear==22.4.25
+          - flake8-comprehensions==3.10.0
+          - flake8-eradicate==1.2.1
+          - flake8-quotes==3.3.1
+          - flake8-simplify==0.19.2
+          - flake8-tidy-imports==4.8.0
+          - flake8-type-checking==1.5.0
+          - flake8-typing-imports==1.12.0
+          - flake8-use-fstring==1.3
+          - pep8-naming==0.12.1
```

```suggestion
          - pep8-naming==0.13.0
```
Released [quite recently](https://github.com/PyCQA/pep8-naming/blob/main/CHANGELOG.rst#0130---2022-06-22) :slightly_smiling_face: